picky-krb/src/constants.rs

```diff
@@ -1,4 +1,5 @@
 pub mod types {
+    /// [Kerberos Message Types](https://datatracker.ietf.org/doc/html/rfc4120#section-7.5.7)
```

Comment:
Triple slash is used to document the next item, so either use
```suggestion
    // [Kerberos Message Types](https://datatracker.ietf.org/doc/html/rfc4120#section-7.5.7)
```
or something like
```suggestion
    //= [Kerberos Message Types](https://datatracker.ietf.org/doc/html/rfc4120#section-7.5.7) =//
```
at your preference

Reply:
Fixed it from all the cases